src/geotiff-optimizer/index.spec.js

```diff
@@ -0,0 +1,34 @@
+process.env.RABBITHOST = '46.4.8.29';
```

Comment:
unsure here. can it be set dynamically?

Reply:
good point. so far we use this approach everywhere https://github.com/klips-project/rabbitmq-worker/blob/main/src/download-file/index.spec.js#L1

Reply:
ok for now. let's change it to a github secret variable or so in follow up pr